packages/types/src/types/ConnectionRequest.ts

```diff
@@ -0,0 +1,5 @@
+export type ConnectionRequest = {
```

Comment:
ConnectRequest and ConnectionRequest, what's the difference? I know that one is the specific object that we get int he qr code flow, but what about the other? Whats its use?

Reply:
`connectRequest` is actually what we have in the extension. The only thing I did was to create the ts type